src/sentry/integrations/github/integration.py

```diff
@@ -319,13 +319,11 @@ def dispatch(self, request: Request, pipeline: Pipeline) -> Response:
             pipeline.bind_state("reinstall_id", request.GET["reinstall_id"])
 
         if "installation_id" in request.GET:
-            organization = self.get_active_organization(request)
-
             # We want to wait until the scheduled deletions finish or else the
             # post install to migrate repos do not work.
             integration_pending_deletion_exists = OrganizationIntegration.objects.filter(
                 integration__provider=GitHubIntegrationProvider.key,
-                organization=organization,
+                organization_id=self.active_organization.id,
```

Comment:
There are a lot of these sorts of filters that take the orm object to query on the relation, but in the future these will not be true foreign keys.

Reply:
There is going to be a pile of changes like this for integrations and sentryapps.